src/hearings/pipes/hearing-answers.pipe.spec.ts

```diff
@@ -169,6 +169,7 @@ describe('HearingAnswersPipe', () => {
       dateRangeEnd: '2022-12-12T09:00:00.000Z',
       firstDateTimeMustBe: '',
     };
+    STATE.hearingRequest.hearingRequestMainModel.hearingDetails.hearingType = 'Choose a date range';
```

Comment:
Could we clone the STATE inside this unit test, then modify it. Otherwise it will polute the test data as this STATE is the intial test data and it's shared everywhere, it will also cause race condition if the tests run randomly.

Reply:
add a line 
```suggestion
const STATE: State = _.cloneDeep(initialState.hearings);
STATE.hearingRequest.hearingRequestMainModel.hearingDetails.hearingWindow = {
      dateRangeStart: '2022-12-12T09:00:00.000Z',
      dateRangeEnd: '2022-12-12T09:00:00.000Z',
      firstDateTimeMustBe: '',
    };
```

Reply:
just need to change line 64
from:
const STATE: State = initialState.hearings;
to:
change that to STATE: State = _.cloneDeep(initialState.hearings);

Then can ignore this place.

Reply:
Done